### ISSUE TITLE
bad compiled rom?

### ISSUE THREAD

**Issue by Mialere** [triforce-percent/triforce-percent#4]
I tried compiling the romhack, and compiled well, but the problem comes when we play the romhack, for some reason when we are going to fight the "runner man", the game crashes, and whe we tried to enter the unicornfountain, we are translated to the potion store

also, checking the scene folder of the git, there isn't any .zmap or .zscene files in the unicornfountain folder

**Comment by sauraen** [comment]
This is intended. The release only contains content which we created ourselves, and no Nintendo copyrighted content. The Running Man boss 3D model is copyrighted, as are many of the textures in all of the custom scenes. So we cannot distribute those items.

**Comment by Mialere** [comment]
I think you go my problem wrong, I mean, I have the original ROM and compile it with it, the problem comes when we play the game, everything goes as intended to the part that we have to fight the running man and more, like the compilation added to the rom in a wrong order, or with missing content, repeate, with the original ROM

also I cheked if I had all the dependecies that are said, but no missing ones, still no idea what's the problem

**Comment by sauraen** [comment]
I guess I didn't explain it clearly enough. The version you built is correct. We did not release any assets copyrighted by Nintendo. So for example, we did not release the 3D model for the Running Man. It isn't in the repo and it won't be in the ROM you built. So when you get to that point in the game, it crashes because the file it needs is missing. It's the same for the custom scenes: they contain Nintendo-copyrighted textures, so we did not distribute them, so they aren't in the version you build, so when you get to them they are missing and you get some other scene.

**Comment by Mialere** [comment]
so, what is this repo functionality at the end?

**Comment by sauraen** [comment]
Do you mean "what was the purpose of releasing this repo" or "what is the list of what things are released in the repo vs. what are missing"?

**Comment by Mialere** [comment]
> Do you mean "what was the purpose of releasing this repo" or "what is the list of what things are released in the repo vs. what are missing"?

the first one

**Comment by sauraen** [comment]
- Why not?
- To help prove that Triforce% was done with ACE rather than just a modded ROM.
- So that future romhack authors and other projects can use the things we made and build on our work.
- People would demand the ability to play Triforce% for themselves, despite the project being an ACE performance. But, we can't release everything due to copyright. So, we released everything we could, and people can play a good chunk of it.
